db/schema.rb

```diff
@@ -58,6 +58,7 @@
     t.jsonb "healthintent_import_data"
     t.string "nhs_number"
     t.date "date_of_birth"
+    t.boolean "has_covid"
```

Comment:
Can we change the field name to `has_covid_symptoms`? I think there may be a big difference and easily misinterpreted.